src/joystream-lib/api.ts

```diff
@@ -65,20 +67,20 @@ export class JoystreamJs {
   }
 
   // if needed these could become some kind of event emitter
-  public onNodeConnectionUpdate?: (connected: boolean) => unknown
+  public onNodeConnectionUpdate?: (connected: boolean) => void
```

Comment:
Do we need this as a class field anymore now that constructor gets it from the argument?